core/src/main/java/com/linecorp/armeria/server/logging/AccessLogComponent.java

```diff
@@ -93,16 +105,90 @@ public Object getMessage(RequestLog log) {
         }
     }
 
+    /**
+     * A timestamp component of a log message.
+     */
+    class TimestampComponent implements AccessLogComponent {
+
+        @VisibleForTesting
+        static final DateTimeFormatter defaultDateTimeFormatter =
+                DateTimeFormatter.ofPattern("dd/MMM/yyyy:HH:mm:ss Z", Locale.ENGLISH);
+        @VisibleForTesting
+        static final ZoneId defaultZoneId = ZoneId.systemDefault();
+
+        static boolean isSupported(AccessLogType type) {
+            return type == AccessLogType.REQUEST_TIMESTAMP;
+        }
+
+        private final boolean addQuote;
+        private final DateTimeFormatter formatter;
+
+        @Nullable
+        private static final Map<String, DateTimeFormatter> predefinedFormatters =
```

Comment:
- `predefinedFormatters` seems to be non-null.
- Could be placed before the member fields?

Reply:
Ooops!! Sorry, that's my fault. Thank you!